build_all_vs2017.bat

```diff
@@ -0,0 +1,30 @@
+@echo off
```

Comment:
Please remove this batch file and the `vswhere.exe` binary, for two reasons: (1) I recently took pains to get rid of unnecessary build scripts, I wouldn't want to go back to a batch file. (2) Fixing the build is unrelated to the issue you're trying to solve.

If you have problems building Moq, then please raise a new issue so we can look at it.